lab-padget/app/view/gallery/gallery.html

```diff
@@ -0,0 +1,3 @@
+<section>
+  <h1>This is the gallery for the app</h1>
```

Comment:
could test out bindings by passing in your {{ galleryCtrl.propertyName }} if you'd like.